src/script/lua_api/l_mainmenu.cpp

```diff
@@ -838,28 +838,27 @@ int ModApiMainMenu::l_get_mainmenu_path(lua_State *L)
 }
 
 /******************************************************************************/
-bool ModApiMainMenu::isMinetestPath(std::string path)
+bool ModApiMainMenu::mayModifyPath(std::string path)
```

Comment:
can you fix the function signature which is wrong in the original code to `const std::string &path`

Thanks

Reply:
done